Makefile

```diff
@@ -71,8 +71,8 @@ build-linux:
 build-windows:
 	@$(MAKE) build GOOS=windows
 
-image:
-	@$(MAKE) build-linux
+image: release
+	mv $(RELEASE_DIR)/opa_envoy_linux_$(GOARCH) .
```

Comment:
`make release` may produce a binary owned by root since it was built inside of Docker...perhaps we could just modify the Dockerfile to copy from the release directory instead of the root directory--that would avoid the `mv`.

Reply:
I'll do that, the path I started on led to a mess, though. I'll clean it up tomorrow. We don't need multiple Dockerfiles, there's only one thing built here. 🤔